src/test/scala/com/wix/mysql/EmbeddedMysqlTest.scala

```diff
@@ -16,7 +16,7 @@ class EmbeddedMysqlTest extends IntegrationTest {
     "start with default values" in {
       val config = aMysqldConfig(Version.v5_7_latest).build
 
-      mysqld = anEmbeddedMysql(config).start
+      val mysqld = withStop(anEmbeddedMysql(config).start)
```

Comment:
I would memoize MySql instance in factory method. And drop ```withStop```
I mean instead of calling imported ```anEmbeddedMysql``` I would create a factory method in ```IntegrationTest```